locales/ta/data.js

```diff
@@ -1,17 +1,16 @@
 export default {
-    question: `Do you have verified data and want to add to the database?`,
-    sourceType: `Source, Type, Delivery`,
-    medicineInjection: `Medicine/Injection/Chemist`,
-    medicineDes: `Distributor, Category, Address`,
-    hospitalDes: `District, Address, Contact`,
-    helplineDes: `Category, Phone`,
-    doctorTele: `Doctor/Telemedicine/Consultation`,
-    doctorDes: `Specialisation, City, Contact`,
-    supportGroup: `Support Groups (WhatsApp, Telegram etc)`,
-    supportDes: `Platform, Category, Contact`,
-    applyVol: `Apply as volunteer`,
-    applyDes: `Skills, Contact`,
-    govtContact: `Government Contact`,
-    govtDes: `Source, City, Contact`,
-    fillTheForm: `Fill the Form`
+    கேள்வி: `உங்களிடம் சரிபார்க்கப்பட்ட தரவு இருக்கிறதா மற்றும் தரவுத்தளத்தில் சேர்க்க விரும்புகிறீர்களா?`,    sourceType: `Source, Type, Delivery`,
```

Comment:
@anupraman2000 you should not translate the key

Reply:
Gotta update the prev PR

Reply:
prev one seemed ok. I checked
https://github.com/coronasafe/life/pull/123/files

Reply:
Because I made the changes , there was 1 key translated,

Reply:
oh got it.. will check the next one to make sure its done right